src/main/java/net/shadowmage/ancientwarfare/npc/tile/TileTownHall.java

```diff
@@ -24,6 +24,8 @@
 
 public class TileTownHall extends TileOwned implements IInventory, IInteractableTile {
 
+    public boolean alarmActive = false;

```

Comment:
As a temporary variable, this will be reset at game stop.
Thus would not follow the redstone state until a new block update occur.
Please check if this is intended for your proposal.
Save using NBT methods if necessary.